hcx-apis/src/main/java/org/swasth/hcx/utils/SearchUtil.java

```diff
@@ -46,7 +43,7 @@ public static SearchRequest buildSearchRequest(final String indexName,
 
 
 
-    private static QueryBuilder getQueryBuilder(final SearchRequestDTO dto) throws ParseException {
+    static QueryBuilder getQueryBuilder(final SearchRequestDTO dto) throws ParseException {
```

Comment:
Test private methods using reflection, or it can be tested via public methods

Reply:
done